internal/generator/vector/helpers/helpers.go

```diff
@@ -63,3 +66,17 @@ func ConfigPath(name string, file string) string {
 func SecretPath(secretName string, file string) string {
 	return fmt.Sprintf("%q", filepath.Join("/var/run/ocp-collector/secrets", secretName, file))
 }
+
+func FromVectorSecretWithSecretPath(secretName, key string) string {
+	return fmt.Sprintf("SECRET[%s.%s_%s]", VectorSecretID, FormatComponentID(secretName), FormatComponentID(key))
```

Comment:
There is a helper function called "MakeID" which takes ariable args which should facilitate this  and replace "FormatComponentID"